test/nxos_ssh/mocked_data/test_get_facts/missing_domain/expected_result.json

```diff
@@ -1,10 +1,10 @@
 {
     "hostname": "nxos1",
     "fqdn": "",
-    "model": "NX-OSv", 
+    "model": "NX-OSv Chassis", 
```

Comment:
To be honest, I'm not sure about this, but looking above at `"model": "Nexus7000 C7009"` I understand your reasoning.

Reply:
Oh I just realised @ktbyers has the same changes in #476.

Reply:
@mirceaulinic about model, I take into reference the nxapi output. I've installed a titanium switch (nxos virtual) and try the get_facts on both.

Reply:
Oh yes, @ktbyers didn't link his PR to his issue so I didn't know about it oups.
But it doesn't work the same way, specially for the hostname